### ISSUE TITLE
perf(log): use pino instead of log4js

### ISSUE THREAD

**Issue by AugustinMauroy** [ether/etherpad-lite#6318]
Pino is built for perf and it's nearly have same API whit log4js.

[Pino docs](https://getpino.io/#/)

**Comment by SamTV12345** [comment]
Good idea. We can add it later in 2.x. The problem is that this would be a breaking change as many plugins rely on log4js

**Comment by AugustinMauroy** [comment]
any news on that ? is it still needed ?

**Comment by JohnMcLear** [comment]
Closing due to lack of traction.
